changelog/7760.breaking.rst

```diff
@@ -0,0 +1,3 @@
+:meth:`sunpy.map.GenericMap.contour` now uses ContourPy instead of scikit-image for improved performance and consistency with :meth:`sunpy.map.GenericMap.draw_contours`.
+
+This does mean that both that the returned contours may differ in order and values compared to the scikit-image version and as that additional keyword arguments are passed through to the underlying function, specific keyword arguments may no longer be supported with ContourPy.
```

Comment:
What does this mean? Is this saying that the result of the contour method won't be the same after switching to contourpy? If so, isn't this a breaking change?